Core/include/Acts/Vertexing/FullBilloirVertexFitter.ipp

```diff
@@ -6,6 +6,7 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
```

Comment:
Did you check whether the FullBilloirVertexFitter works better now for spatially close but temporally distinct vertices?

Reply:
I guess we should do this once we have the full Jacobian